modules/basics/src/main/java/com/opengamma/basics/date/ImmutableHolidayCalendar.java

```diff
@@ -51,6 +51,12 @@
 @BeanDefinition(builderScope = "private")
 public final class ImmutableHolidayCalendar
     implements HolidayCalendar, ImmutableBean, Serializable {
+  // optimized implementation of HolidayCalendar
+  // uses an int array where each int represents a month
+  // each bit within the int represents a date, where 0 is a holiday and 1 is a business day
+  // (most logic involves finding business days, finding 1 is easier than finding 0)
```

Comment:
Not sure why finding 1 is easier than finding 0. You'd always use ``(1 << lookupDom0)`` as the mask to isolate the bit you're interested in, AND it with the integer and compare the result with 0.